int32 goal_position
int32 goal_velocity
int32 goal_acceleration
---
bool goal_accepted